modules/wrapper/rc/config.nix

```diff
@@ -55,6 +55,8 @@ in {
         pluginConfigs = entryAfter ["optionsScript"] pluginConfigs;
         extraPluginConfigs = entryAfter ["pluginConfigs"] extraPluginConfigs;
         mappings = entryAfter ["extraPluginConfigs"] keymaps;
+        # FIXME: put this somewhere less stupid
+        footer = entryAfter ["mappings"] (optionalString config.vim.lazy.enable "require('lzn-auto-require').enable()");
```

Comment:
I think you can safely remove this comment, because this doesn't seem stupid. 2 things to do though:
- name it something other than footer
- update the _DAG entries_ manual page accordingly